docs/section4/4-11.md

```diff
@@ -0,0 +1,65 @@
+# Airside System - Rule 4-11  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** True  
+**Rule ID:** 4-11    
+ 
+**Rule Description:** Fan Schedules may be allowed to differ when Section G3.1.1(c) applies, i.e. baseline HVAC system type 3 or 4 has been used for spaces (zones) that differ by more than 40 equivalent full load hrs/week from other spaces (zones) served by the system or when the total peak internal gains that differ by 10 Btu/h·ft2 or more from the average of other HVAC zones served by the system.  
+
+**Rule Assertion:** P-RMR = B-RMR                                           
+**Appendix G Section:** Table G3.1 Section 4 Baseline Exception 3  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem data table   
+
+**Applicability Checks:** 
+
+1. Applies when baseline HVAC system type 3 or 4 has been used for spaces (zones) that differ by more than 40 equivalent full load hrs/week from other spaces (zones) served by the system or when the total peak internal gains that differ by 10 Btu/h·ft2 or more from the average of other HVAC zones served by the system.    
+
+**Function Call:** 
+
+1. get_baseline_system_types()     
+2. get_hvac_system_with_longest_fan_schedule()  
+3. match_data_element()  
+4. get_dict_of_zones_hvac_sys_serving_specific_floor()  
+5. get_list_hvac_systems_associated_with_zone()
+6. get_hvac_zone_list_w_area()
+
+## Rule Logic:  
+**Applicability Check 1:**  
+- Get dictionary for finding baseline system types for hvac systems: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get dictionary with list of zones associated with each HVAC system: `dict_hvac_sys_zones_served_p = get_hvac_zone_list_w_area(P_RMR)`
+- For each zone in the P_RMR: `for zone_p in P_RMR...Zone:`
+    - Reset schedule mismatch variable which is used to track if the schedules mismatches: `schedule_mismatch = FALSE`
+    - Get hvac system with the longest fan operating schedule in the proposed to compare to the baseline fan schedule: `hvac_p = get_hvac_system_with_longest_fan_schedule_p(zone_p.id, P_RMR)`
```

Comment:
@mkarpman i'm wondering if instead off the longest scehdule, we should be getting the merged schedule. In your example of a zone having a fan schedule from 0-12, and another zone having 13-24, the baseline should have a fan schedule of 0-24, correct? so should this be a merged schedule instead of the longest schedule?